src/plots/Vector/avtVectorPlot.C

```diff
@@ -51,18 +51,18 @@
 //    Use glyph's output port when creating avtVectorGlyphMapper.
 //
 // ****************************************************************************
```

Comment:
You have removed the whitespace between comment blocks and functions and the extra line between function. This particular whitepace is standard in our style, can you please fix?